app/templates/static/maintaining.static.pug

```diff
@@ -0,0 +1,33 @@
+block content
+  body(style="background-image: url(/images/pages/maintaining/background.jpg); background-size: 100vw; display: flex; flex-direction: column; align-items: center;", onload="onLoad()")
+    img(id='logo', src="/images/pages/base/logo.png")
+    h3(id='title', style="margin-top: 20vh;font-size:48px;color: #317EAC")
+    span(id='msg', style="margin-top: 30vh;font-size:24px;")
+
+block script-start
+  script(src='/maintaining_time')
+  script.
+    function onLoad() {
+      time = maintainingHours
+      duration = null
+      if(!isNaN(+time))
+        duration = +time
+      if(chinaUx){
+        document.querySelector('#logo').src = '/images/pages/base/logo-cn.png'
+        maintainingTitle = '网站升级维护中...'
+        maintainingMsg = '感谢您的耐心等待'
+        if(duration) {
+          maintainingMsg = '预计耗时 ' + duration + ' 小时，' + maintainingMsg
+        }
+      }
+      else{
+        maintainingTitle = 'Website is maintaining...'
+        maintainingMsg = 'Thanks for your patience'
+        if(duration){
+          maintainingMsg = 'It may takes ' + duration + ' hours, '  + maintainingMsg
+        }
```

Comment:
```suggestion
        maintenanceTitle = 'Website Maintenance In Progress'
        maintenanceMsg = 'Thanks for your patience.'
        if(duration){
          maintainingMsg = 'It may take up to ' + duration + ' hours. '  + maintainingMsg
        }
```

Reply:
Here and on the server, it would be cleaner to write everything as `maintenance` instead of `maintaining`.